pyproject.toml

```diff
@@ -29,11 +29,11 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.9"
-tibanna = "^3.3.1"
-dcicutils = "^7.0.0"
-boto3 = "^1.9.0"
-botocore = "^1.12.1"
+python = ">=3.7,<3.12"
+tibanna = "4.0.0.1b1"
+dcicutils = "7.12.0.1b1"
+boto3 = "^1.28.51"
```

Comment:
A good idea to include the corresponding `boto3-stubs` and `botocore-stubs` in dev-dependencies. I think those are supposed to have hinting or some other kind of debugging info.

Reply:
Done